src/library-authoring/common/context/LibraryContext.tsx

```diff
@@ -26,11 +26,16 @@ export type LibraryContextData = {
   libraryData?: ContentLibrary;
   readOnly: boolean;
   isLoadingLibraryData: boolean;
-  /** The ID of the current collection/component/unit, on the sidebar OR page */
+  /** The ID of the current collection/unit/subsection/section, on the sidebar OR page */
   collectionId: string | undefined;
   setCollectionId: (collectionId?: string) => void;
   unitId: string | undefined;
   setUnitId: (unitId?: string) => void;
+  subsectionId: string | undefined;
+  setSubsectionId: (subsectionId?: string) => void;
+  sectionId: string | undefined;
+  setSectionId: (sectionId?: string) => void;
+  containerId: string | undefined;
```

Comment:
To do: can we avoid adding all of these and just have `containerId` and `setContainerId` ?

Reply:
Addressed with 5cecfa8ab9fb034be314fbb17db5ddff619f9661